src/pages/listing/listing.css

```diff
@@ -118,170 +95,164 @@
 }
 
 .currPage {
-    font-size: 18px;
     vertical-align: middle;
+    font-size: 18px;
 }
 
 .add {
     composes: add from buttons_css;
-    composes: ellipsisOverflow;
 }
 
 .showingResults {
     composes: global(pure-u-1-1);
 }
 
-.listContainer {
-    composes: global(pure-g);
+.entriesContainer {
+    composes: lightSepTop from layout_css;
 
     width: 100%;
     margin: 0 auto;
 }
 
-.list {
-    list-style: none;
-    margin: 0 auto;
-    padding: 0 0 4em 0;
+.table {
     width: 100%;
+    margin: 0 0 4em 0;
 }
 
-.listHeader {
-    composes: global(pure-g);
+.col {
+    padding: 0.5em pad;
+    max-width: 0; /* needed for ellipsis overflow on <td> */
 
-    border-bottom: solid 1px #AAA;
-    padding-top: 0.6em;
-    font-size: 1.1em;
+    text-align: center;
 }
 
-.listHeader .listCell {
-    padding-top: 0;
-}
+.colName {
+    composes: col;
 
-.listCell {
-    composes: ellipsisOverflow;
+    width: 40%;
 
-    margin: 0;
-    padding: 8px 2%;
-    line-height: 24px;
-}
+    border-left: solid 3px color_draft;
 
-.listCol1 {
-    composes: global(pure-u-9-24);
-    composes: listCell;
+    text-align: left;
 }
 
-.listCol2 {
-    composes: global(pure-u-4-24);
-    composes: listCell;
-}
+.colState {
+    composes: col;
 
-.listCol3 {
-    composes: global(pure-u-4-24);
-    composes: listCell;
-}
+    width: 16%;
 
-.listCol4 {
-    composes: global(pure-u-4-24);
-    composes: listCell;
+    text-align: left;
 }
 
-.listCol5 {
-    composes: global(pure-u-3-24);
-    composes: listCell;
+.colDate {
+    composes: col;
 
-    text-align: right;
+    width: 16%;
 }
 
-.listHeader,
-.list .item:nth-child(odd) {
-    background: rgba(0,0,0,0.025);
-}
+.colActions {
+    composes: col;
 
-.item {
-    composes: global(pure-g);
+    width: 12%;
 
-    position: relative;
+    text-align: right;
+}
 
-    padding-left: 4px;
+.row {
+    cursor: pointer;
     border-bottom: solid 1px #CCC;
 
-    transition: background-color 0.2s ease-in;
+    transition: background-color 0.1s ease-in;
+}
+
+.row:nth-child(even) {
+    background: rgba(0, 0, 0, 0.025);
 }
 
-.item:hover {
+.row:hover {
     background: #DFDFDF;
+}
+
+.published_at {
+    composes: row;
 
-    transition: background-color 0.2s ease-in;
+    td:first-child {
```

Comment:
Actually, pretty sure that class could be applied to the `td` instead of the `tr` (thus no longer necessitating the descendant).